content/en/docs/Responses/AlbumID3.md

```diff
@@ -11,19 +11,55 @@ description: >
 {{< tab header="**Example**:" disabled=true />}}
 {{< tab header="OpenSubsonic" lang="json">}}
 {
-  "id": "ad0f112b6dcf83de5e9cae85d07f0d35",
-  "name": "8-bit lagerfeuer",
-  "artist": "pornophonique",
-  "year": 2007,
-  "coverArt": "al-ad0f112b6dcf83de5e9cae85d07f0d35_640a93a8",
-  "starred": "2023-03-22T01:51:06Z",
-  "duration": 1954,
-  "playCount": 97,
-  "created": "2023-03-10T02:19:35.784818075Z",
-  "artistId": "91c3901ac465b9efc439e4be4270c2b6",
-  "songCount": 8,
-  "played": "2023-03-28T00:45:13Z",
-  "userRating": 4
+    "id": "ad0f112b6dcf83de5e9cae85d07f0d35",
+    "name": "8-bit lagerfeuer",
+    "artist": "pornophonique",
+    "year": 2007,
+    "coverArt": "al-ad0f112b6dcf83de5e9cae85d07f0d35_640a93a8",
+    "starred": "2023-03-22T01:51:06Z",
+    "duration": 1954,
+    "playCount": 97,
+    "genre": "Hip-Hop",
+    "created": "2023-03-10T02:19:35.784818075Z",
+    "artistId": "91c3901ac465b9efc439e4be4270c2b6",
+    "songCount": 8,
+    "played": "2023-03-28T00:45:13Z",
+    "userRating": 4,
+    "albumLabel": "Sony",
+    "musicBrainzId": "189002e7-3285-4e2e-92a3-7f6c30d407a2",
+    "genres": [
+        "Hip-Hop",
+        "East coast"
+    ],
+    "artists": [
+        {
+            "id": "ar-1",
+            "name": "Artist 1"
+        },
+        {
+            "id": "ar-2",
+            "name": "Artist 2"
+        }
+    ],
+    "displayArtist": "Artist 1 feat. Artist 2",
```

Comment:
Do we need a `displayArtist` field? Legacy subsonic clients won't use any of the new fields and a modern OpenSubsonic client has the full artist info through `artists` and could come up with whatever display artist name they want (like Artist 1, Artist 2 with each being a clickable hyperlink)